plans/dockerregistry.tf

```diff
@@ -0,0 +1,119 @@
+resource "azurerm_resource_group" "dockerregistry" {
+  name     = "${var.prefix}dockerregistry"
+  location = "${var.location}"
+  tags {
+    env = "${var.prefix}"
+  }
+}
+
+resource "azurerm_storage_account" "dockerregistry" {
+    name                = "${var.prefix}dockerregistry"
+    resource_group_name = "${azurerm_resource_group.dockerregistry.name}"
+    location            = "${var.location}"
+    account_type        = "Standard_GRS"
+    tags {
+        env = "${var.prefix}"
+    }
+}
+
+resource "azurerm_template_deployment" "dockerregistry"{
```

Comment:
Per our in-person discussion, this should use the [depends_on](https://www.terraform.io/docs/configuration/resources.html#explicit-dependencies) parameter to explicitly depend on the resources defined above such that they are created before the template is submitted for deployment